lighthouse-core/lib/stack-packs.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const stackPacks = require('@lighthouse/stack-packs');
+
```

Comment:
I think we want something like this

```js
const stackPacksToInclude = [{
  packId: 'wordpress'
  npmOrIconName: 'wordpress',
}];
```

Below when we loop over the discovered Stacks, we only continue adding a stack pack if `npmOrIconName` matches the [Library](https://github.com/GoogleChrome/lighthouse/blob/81a7589efb12d73c7830ba12376b39f1b377c827/types/artifacts.d.ts#L87). (We have to add icon in there.)

Doing this allows the stack-pack repo to develop new packs without worrying that we're including them before they're ready. Seem good @housseindjirdeh ?